### DISCUSSION TITLE
What's the training data of your pretrained model?

### DISCUSSION THREAD

**Discussion by imzhangyd** [BBQuercus/deepBlink#138] [General]
Hi, I'm working on the single particle tracking on ISBI2012 Particle Tracking Challenge. The challenge offered 2 dataset, one is called "training image data" for designing method, another is called "challenge data" for evaluating different methods. I'm wondering which one dataset is used for training your pretrained model "microtubule.h5","vesicle.h5","receptor.h5". If "challenge data" is not used, I'd like to use your pretrained model directly.

**Comment by BBQuercus** [comment]
Hey. If I recall correctly we only used the training image data. The image files used for our test splits are described in the csv files on figshare (https://figshare.com/articles/dataset/Datasets/12958037).

**Comment by imzhangyd** [comment]
@BBQuercus Thanks for your quick reply!
I have compared your test images with the two dataset from ISBI. The result is that the dataset used by deepBlink is from "challenge data", not "training image data". So I should have to retrain from scratch. Would you have some suggestions about the train config?
Thank you very much!

**Comment by BBQuercus** [comment]
I see - for all published models we used the default configuration you get from `deepblink config`. I would suggest you start with that and see if it works well enough for your use case before doing any further optimisations.

**Comment by imzhangyd** [comment]
Thanks for your suggenstion. I'll try it.
